### PR TITLE
Power d adaptations

### PR DESCRIPTION
**Pull request by CarlosEpia**

### CODE REVIEW

File: etrago/appl.py [hunk 1]
```diff
@@ -455,13 +455,10 @@ def run_etrago(args, json_path):
 
     # import network from database
     etrago.build_network_from_db()
-
+
```
Comment (ClaraBuettner) [inline etrago/appl.py:458]: This line looks like you should run black

File: etrago/appl.py [hunk 2]
```diff
@@ -455,13 +455,10 @@ def run_etrago(args, json_path):
...
     # import network from database
     etrago.build_network_from_db()
-
+    
     # adjust network regarding eTraGo setting
     etrago.adjust_network()
 
-    # ehv network clustering
-    etrago.ehv_clustering()
-
```
Comment (ClaraBuettner) [inline etrago/appl.py:464]: I guess that was not intended, right?

File: etrago/appl.py [hunk 3]
```diff
@@ -155,7 +155,7 @@
     },
     # Simplifications:
     "branch_capacity_factor": {"HV": 0.5, "eHV": 0.7},  # p.u. branch derating
-    "load_shedding": True,  # meet the demand at value of loss load cost
+    "load_shedding": False,  # meet the demand at value of loss load cost
```
Comment (ClaraBuettner) [inline etrago/appl.py:158]: Changes in the args are not needed

File: etrago/appl.py [hunk 4]
```diff
@@ -74,7 +74,7 @@
         "threads": 4,
     },
     "model_formulation": "kirchhoff",  # angles or kirchhoff
-    "scn_name": "eGon2035",  # scenario: eGon2035 or eGon100RE
+    "scn_name": "status2019",  # scenario: eGon2035 or eGon100RE
```
Comment (ClaraBuettner) [inline etrago/appl.py:77]: It would be good to add "status2019" to the scenarios in the comment

File: etrago/tools/utilities.py [hunk 5]
```diff
@@ -2336,7 +2336,9 @@ def check_args(etrago):
 
     """
 
-    names = ["eGon2035", "eGon100RE", "eGon2035_lowflex", "eGon100RE_lowflex"]
+    names = ["eGon2035", "eGon100RE", "eGon2035_lowflex", "eGon100RE_lowflex",
+             "status2019"]
+
```
Comment (ClaraBuettner) [inline etrago/tools/utilities.py:2341]: "Black" is complaining about this change, I guess you should run it again

**Comment by CarlosEpia** [comment]
All the comments were addressed in [bfc4d0a](https://github.com/openego/eTraGo/pull/648/commits/bfc4d0ae2d40dfee804836ffccd4fb0a5fb58cfd)

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**